src/state/types.ts

```diff
@@ -95,7 +104,10 @@ export type StageEventType =
   | "habitatUnlock"
   | "cardBuy"
   | "gameLoss"
-  | "gameWin";
+  | "gameWin"
+  | "policy_specialDraw"
+  | "policy_fundingIncrease"
+  | "policy_climateChange";
```

Comment:
Lets invert that. Those types should come from @/state/machines/expansion/index.ts

```ts
| ExtPackEvent;
```